transport_nantes/topicblog/views.py

```diff
@@ -619,6 +625,105 @@ def form_valid(self, form):
                 send_record.save()
 
         return super().form_valid(form)
+
+
+class TopicBlogSelfSendView(PermissionRequiredMixin, LoginRequiredMixin,
+                            SendableObjectMixin, BaseFormView):
+    """Allow authorised users to send a not published sendable TopicBlog
```

Comment:
PEP-8 etc: one-line, blank line, then body, all respecting line length limits.

Reply:
and: that would be an "unpublished".  But, more generally, the point is that you can send by id whether it's published or not.